stories/Button.tsx

```diff
@@ -0,0 +1,47 @@
+import './button.css'
+
+export interface ButtonProps {
+  /**
+   * Is this the principal call to action on the page?
```

Comment:
this file / these comments are part of the storybook defaults i assume? nice

Reply:
yup! I figured I would leave them for now as a good reference